pom.xml

```diff
@@ -59,7 +59,7 @@
         <jdk-version>1.7</jdk-version>
 
         <!-- versions -->
-        <zookeeper-version>3.5.3-beta</zookeeper-version>
+        <zookeeper-version>3.5.4-beta</zookeeper-version>
```

Comment:
-1 one this. Let's leave dep/version changes for a separate PR

Reply:
This is required. Zk#close(timeout) is only in 3.5.4.
I can log a separate JIRA and create a simple PR

Reply:
No problem - just checking.